### PR TITLE
add more troubleshooting (back)

### PR DESCRIPTION
**Pull request by Gruetzig**

### CODE REVIEW

File: _pages/en_US/include/exploit-usm.txt [hunk 1]
```diff
@@ -9,3 +9,5 @@ In this section, you will enter Safe Mode (a feature available on all 3DS family
 1. When asked "Would you like to configure Internet settings?", select "Yes"
 1. On the following menu, navigate to `Connection 1` -> `Change Settings` -> `Next Page (right arrow)` -> `Proxy Settings` -> `Detailed Setup` ([image](/images/screenshots/usm/safemode_highlighted.png))
 1. If the exploit was successful, your device will have booted into SafeB9SInstaller
+  + If your device instead freezes on a white screen, hold the POWER button until it turns off, then retry this section
+  + If your device instead freezes on a red screen, you are missing usm.bin from the root of your SD card
```
Comment (lifehackerhansol) [inline _pages/en_US/include/exploit-usm.txt:13]: wrap `usm.bin` in backticks

File: _pages/en_US/troubleshooting.txt [hunk 2]
```diff
@@ -420,6 +421,32 @@ If this is unsuccessful, try using another SD card.
...
+
+The date is set incorrectly. \
+To set it correctly, follow these steps: 
+1. Select the System Settings icon on the HOME Menu, and tap Open.
+1. Tap Other Settings.
+1. Tap Date & Time.
+1. Tap Today's date.
+1. Tap Up/Down Arrows to set the correct Day, Month and Year.
+1. Select OK to confirm.
+When done, go back to [Section III of Homebrew Launcher (super-skaterhax)](homebrew-launcher-(super-skaterhax)#section-iii---super-skaterhax).
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:437]: we don't need this "go back to previous page" thing imo. It'll make both translations awkward and the whole point of the troubleshooting page is that you will do troubleshooting and just go back to where you were

File: _pages/en_US/homebrew-launcher-(super-skaterhax).txt [hunk 3]
```diff
@@ -64,6 +63,9 @@ If you haven't already, make sure you have a working Internet connection set up
 1. Press (Home) to return to the HOME Menu, then immediately press (A) to launch the browser again
 1. Select the "GO GO!" button on the top screen
   + If any prompts appear, approve all of them
+  + If your device freezes on a yellow screen, hold the POWER button until it turns off, then retry this section
+  + If your device freezes on a red screen, hold the POWER button until it turns off, redo step Step 3 of Section II, then retry this section
+    + If you get another error, [follow this troubleshooting guide](troubleshooting#homebrew-launcher-super-skaterhax)
```
Comment (lifehackerhansol) [inline _pages/en_US/homebrew-launcher-(super-skaterhax).txt:68]: If I recall correctly, level 2 nested subbullets do not work in Jekyll. Make it line up with the other bullets

File: _pages/en_US/troubleshooting.txt [hunk 4]
```diff
@@ -420,6 +421,32 @@ If this is unsuccessful, try using another SD card.
...
+If the problem persists and you have a system version ending in E, and the language set to English, change the language to any other language **temporarily**. \ 
+You can revert this change when you reach the Finalizing Setup.
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:439]: `s/the //`

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]

File: _pages/en_US/homebrew-launcher-(super-skaterhax).txt (hunk 3 above)
Comment (Gruetzig) [reply _pages/en_US/homebrew-launcher-(super-skaterhax).txt:68]: accident

**Comment by lifehackerhansol** [comment]
Merged to HEAD

**Pull request closed without merging**